JavaScript/JavaScript.sublime-syntax

```diff
@@ -1836,43 +1836,43 @@ contexts:
           | (\.) {{dec_digit}}+ (?:{{dec_exponent}})?
         ){{identifier_break}}
       captures:
-        0: meta.number.value.js constant.numeric.float.decimal.js
+        0: meta.number.float.decimal.js constant.numeric.value.js
         1: punctuation.separator.decimal.js
         2: punctuation.separator.decimal.js
       pop: true
 
     # integers
     - match: (0)({{dec_digit}}+){{identifier_break}}
       captures:
-        1: meta.number.base.js constant.numeric.integer.octal.js invalid.deprecated.numeric.octal.js
-        2: meta.number.value.js constant.numeric.integer.octal.js invalid.deprecated.numeric.octal.js
+        1: meta.number.integer.octal.js constant.numeric.base.js invalid.deprecated.numeric.octal.js
+        2: meta.number.integer.octal.js constant.numeric.value.js invalid.deprecated.numeric.octal.js
       pop: true
 
     - match: (0[Xx])({{hex_digit}}*)(n)?{{identifier_break}}
       captures:
-        1: meta.number.base.js constant.numeric.integer.hexadecimal.js
-        2: meta.number.value.js constant.numeric.integer.hexadecimal.js
-        3: meta.number.suffix.js constant.numeric.integer.hexadecimal.js
+        1: meta.number.integer.hexadecimal.js constant.numeric.base.js
```

Comment:
Can you `scope: meta.number.integer.hexadecimal.js` and leave just the `constant`s broken out in the capture groups?

Reply:
Done.